std/assembly/array.ts

```diff
@@ -13,6 +13,9 @@ import {
   weakHeapSort
 } from "./internal/array";
 
+import { itoa } from "./internal/itoa";
+import { dtoa } from "./internal/dtoa";
```

Comment:
Could maybe make this `internal/number` holding all the number compatibility code

Reply:
Sure, make sense

Reply:
I think `parse<T>` should be also move to `internal/number`. WDYT?